src/pfe/portal/modules/Templates.js

```diff
@@ -23,18 +23,21 @@ const log = new Logger('Templates.js');
 const DEFAULT_REPOSITORY_LIST = [
   {
     url: 'https://raw.githubusercontent.com/kabanero-io/codewind-templates/master/devfiles/index.json',
-    description: 'Standard Codewind templates',
+    description: 'Codewind is an extension to develop in containers with an IDE or language of your choice.',
```

Comment:
Hi, @jcockbain96, I just met with some coworkers today about this description. Could we please change it to this?
Codewind project templates help you create containerized projects for various runtimes.

Reply:
Fixed to resolve for DCUT